includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -183,6 +183,17 @@ public function sanitize() {
 			$elements[] = $element;
 		}
 
+		// If 'width' attribute is present for 'col' tag, convert to proper CSS rule.
+		foreach ( $this->dom->getElementsByTagName( 'col' ) as $col ) {
+			$width_attr = $col->getAttribute( 'width' );
+			if ( ! empty( $width_attr ) && ! strpos( $width_attr, '*' ) ) {
+				strpos( $width_attr, '%' )
+					? $col->setAttribute( 'style', 'width: ' . $width_attr )
+					: $col->setAttribute( 'style', 'width: ' . $width_attr . 'px' );
```

Comment:
Using a ternary expression as a control statement is usually discouraged. I recommend doing something like this:

```php
$width_style = "width: $width_attr";
if ( is_numeric( $width_attr ) ) {
    $width_attr .= 'px';
}
$col->setAttribute( 'style', $width_style ); // @todo Still need to be wary of there being an existing style attr.
```

Reply:
Nice. Will do.